Refresh keyword

jcaille opened the pull request:
* Added a `refresh` keyword to manually trigger a refresh of a plugin.
* Improved the UI when a plugin is refreshing by displaying the short string `"Updating ..."` in place of the (now out of date) plugin information.
* Updated the documentation to reflect those changes

In the code, I now keep a reference to the timer triggering the next automatic refresh, invalidating it when a manual refresh is requested.
Before launching a new task, I set the content to `"Updating..."` and manually trigger a menuItem rebuild.

jtokoph commented:
Does this show "Updating..." for automatic refreshes too? That could get pretty wild for plugins that refresh frequently.

jcaille commented:
Good point, at the moment it does. 
Do you think the `Updating...` should only be displayed when a manual refresh is requested (so as to show that the user's action has been taken into account ?)

matryer commented:
Yes, "Updating..." should only show when it's manual, otherwise it's nice that it just updates. Especially since, when it hasn't changed, it doesn't attract any attention.

jcaille commented:
I've updated the PR :

* "Updating ..." should only be displayed when a manual refresh has been triggered
* Moved the documentation for new keyword to `Plugins/Readme.md`
